core/chains/evm/log/broadcaster.go

```diff
@@ -79,7 +79,7 @@ type (
 
 		ethSubscriber *ethSubscriber
 		registrations *registrations
-		logPool       *logPool
+		logPool       iLogPool
```

Comment:
What is the benefit of using an interface here?

Reply:
When we add unit-tests for this broadcaster.go file, and want to mock out the logPool dependency, we can't do it unless we had an interface here, which we then populate with a Mock.
This PR didn't add unit-tests for broadcaster.go, but when we do, this will be needed.

It is also the pattern to use for future. Whenever you want to unit-test a file, you have to refer to all your dependencies via interfaces, and not directly embed the struct.

Reply:
Why not use an actual pool?

Reply:
You mean not the interface, but the struct directly?
Works for now, until we add unit-tests for this file, and want to mock out the logPool.